workspace/notebook/appeal_s78_curated_mipins.json

```diff
@@ -206,24 +216,68 @@
 					"FROM odw_harmonised_db.sb_appeal_s78 AS AH\n",
 					"where AH.lpaCode NOT in ('Q9999', 'Q1111')"
 				],
-				"execution_count": 28
+				"execution_count": 39
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"spark = SparkSession.builder.getOrCreate()\n",
+					"view_df = spark.sql(\"SELECT * FROM odw_curated_db.vw_appeals_s78_curated_mipins\")"
+				],
+				"execution_count": 40
+			},
+			{
+				"cell_type": "code",
+				"metadata": {
+					"collapsed": false
+				},
+				"source": [
+					"# prepare dataframe convert UTC to Day light savings for all timestamp fields\n",
```

Comment:
Thinking, could it be simpler to define this as a loop instead, since the arguments to `from_utc_timestamp` are the same for each `withColumn` call ?

Reply:
Something like

```
col_list = [
    "caseSubmittedDate",
    "caseCreatedDate",
    ...
]
final_view_df = view_df
for column in col_list:
    final_view_df = final_view_df.withColumn(column, from_utc_timestamp(column, "Europe/London")
```